packages/liveblocks-core/src/__tests__/Poller.test.ts

```diff
@@ -0,0 +1,146 @@
+import { makePoller } from "../lib/Poller";
+
+// Mock timers using Jest
+jest.useFakeTimers();
+
+describe("Poller", () => {
+  beforeEach(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.clearAllTimers();
+  });
+
+  test("should start the poller when condition is true", async () => {
+    const callback = jest.fn();
+    const poller = makePoller(callback, 1000); // 1000ms interval
+
+    poller.enable(true);
+
+    // Fast-forward until all timers have been executed
+    await jest.advanceTimersByTimeAsync(1000);
+
+    // Expect the callback to be called after the interval
+    expect(callback).toHaveBeenCalledTimes(1);
```

Comment:
Might be nice to add an assertion that after 500ms, the callback has not been invoked yet.